src/components/projections/Redact/SaveButton.tsx

```diff
@@ -44,11 +48,30 @@ const SaveButton = ({
         <Button
             disabled={formActive || saving}
             onClick={() => {
+                if (previousStrategy === strategy) {
+                    logRocketEvent(CustomEvents.COLLECTION_SCHEMA, {
+                        operation: 'redact',
+                        pointer,
+                        previousStrategy,
+                        strategy,
+                    });
+
+                    closeDialog();
+                    return;
+                }
+
                 setSaving(true);
 
                 updateRedactionAnnotation(pointer, strategy)
                     .then(
                         (response) => {
+                            logRocketEvent(CustomEvents.COLLECTION_SCHEMA, {
+                                operation: 'redact',
+                                pointer,
+                                strategy,
+                                dataExists: Boolean(response?.data?.[0]),
+                            });
```

Comment:
This is a nitpick... but I think we should store off the `dataExists` into a variable and use it here and down below in the if. That way we are logging the literal variable we use down below to control if we set collection data or not.